conf/distro/nilrt.inc

```diff
@@ -141,6 +142,10 @@ NILRT_FEEDS_URI_RELEASE            ?= "${NILRT_FEEDS_URI}/${NILRT_FEED_NAME}"
 NILRT_MACHINE_FEED_URI_x64         ?= "${NILRT_FEEDS_URI_RELEASE}/x64"
 NILRT_MACHINE_FEED_URI_xilinx-zynq ?= "${NILRT_FEEDS_URI_RELEASE}/arm"
 
+NILRT_MAIN_FEED_VERSION            ?= "2020.7"
+NILRT_MACHINE_FEED_URI_nimain      ?= "${NILRT_FEEDS_URI}/${NILRT_MAIN_FEED_VERSION}/ni-main"
+NILRT_MACHINE_FEED_URI_lv2020      ?= "${NILRT_FEEDS_URI}/${NILRT_MAIN_FEED_VERSION}/ni-lv2020"
```

Comment:
[Follow-up Item]
Likewise, I think we have an ni-lv2021/ feed now, which needs to be updated here and in the `IPK_FEED_URIS` variable later in this file.